BUILD.bazel

```diff
@@ -46,13 +46,13 @@ go_binary(
 
 # Create the non-root user, "bazelremote", so the container can run with reduced
 # privileges.
-BAZEL_REMOTE_USER = "bazelremote"
+BAZEL_REMOTE_USER_NAME = "bazelremote"
```

Comment:
nitpick: please change this to bazel-remote.

Reply:
Done